build.sh

```diff
@@ -111,45 +139,79 @@ inject_metadata () {
 # Main entry point.
 # ------------------------------------------------
 
-# Make sure we start from a clean slate.
-rm -rf built
-rm -rf downloads_cache
-rm -f master.tar.gz
-rm -f metadata.tar.gz
-
-# Run basic tests.
-echo "Running basic sanity tests on metadata."
-echo "If these fail, then fix whatever is causing them first."
+if [ -n "$ghprbActualCommit" ]
+then
+    echo "Commit hash: $ghprbActualCommit"
+    export COMMIT_CHANGES="`git diff --diff-filter=AM --name-only --stat origin/master`"
+else
+    echo "No commit provided, skipping further tests."
+    exit $EXIT_OK
+fi
 
```

Comment:
I'm not well versed enough with Jenkins to know if there is ever a scenario that we don't have a commit ID. We're actually just using it for an identifier right now.